mobile_packet_verifier/src/burner.rs

```diff
@@ -81,14 +81,34 @@ where
             },
         ) in payer_totals.into_iter()
         {
-            tracing::info!("Burning {total_dcs} DC from {payer}");
+            tracing::info!(%total_dcs, %payer, "Burning DC");
 
-            self.solana
+            if self
+                .solana
                 .burn_data_credits(&payer, total_dcs)
                 .await
-                .map_err(BurnError::SolanaError)?;
+                .is_err()
+            {
+                // We have failed to burn data credits:
+                metrics::counter!("burned", total_dcs, "payer" => payer.to_string(), "success" => "false");
```

Comment:
having to base graph queries off this "success" label seems like it could lead to inaccurate visualizations when you could forget to account for that label and inadvertently be adding both failed and succeeded burn amounts together. might be better to just have two separate metrics @bbalser ?